src/screens/CreateScreen.js

```diff
@@ -26,9 +26,10 @@ const CreateScreen = ({ navigation }) => {
             <Button 
                 title="Add Blog Post"
                 onPress={() => {
-                    addBlogPost(title, content)
-                    navigation.navigate('Index')
-                    }}
+                    addBlogPost(title, content, () => {
```

Comment:
we added a callback function inside of addBlogPost for future reason when saving data in a database